dapr/proto/common/v1/README.md

```diff
@@ -5,7 +5,7 @@ This folder is intended for the Common protos amongst the packages in the `dapr/
 ## Proto client generation
 
 Pre-requisites:
-1. Install protoc version: [v4.24.4](https://github.com/protocolbuffers/protobuf/releases/tag/v4.24.4)
+1. Install protoc version: [v4.25.4](https://github.com/protocolbuffers/protobuf/releases/tag/v4.25.4)
```

Comment:
```suggestion
1. Install protoc version: [v25.4](https://github.com/protocolbuffers/protobuf/releases/tag/v25.4)
```
Shouldn't this be `v25.4`? From what I understand, the 4.X.X would be the language runtime-specific version, but since Go is not included in the compiler and instead used as a plugin, it probably doesn't make sense for us to refer to 4.X.X here.
Please correct me if my understanding is wrong here, I find protobuf versioning to be quite confusing.

Reply:
To be quite honest, I'm not sure.

Reply:
I agree with this assessment although the tags point and align at the same commit references thus making a minimal difference, we should move towards a clearer version/tag consistent with what we already have.